scheduler/test/cook/test/kubernetes/compute_cluster.clj

```diff
@@ -78,10 +78,10 @@
                                                           (atom {}) (atom {}) (atom {}) (atom {}) (atom nil)
                                                           {:kind :static :namespace "cook"} nil 3 nil nil
                                                           (Executors/newSingleThreadExecutor))
-          node-name->node {"nodeA" (tu/node-helper "nodeA" 1.0 1000.0 nil)
-                           "nodeB" (tu/node-helper "nodeB" 1.0 1000.0 nil)
-                           "nodeC" (tu/node-helper "nodeC" 1.0 1000.0 nil)
-                           "my.fake.host" (tu/node-helper "my.fake.host" 1.0 1000.0 nil)}
+          node-name->node {"nodeA" (tu/node-helper "nodeA" 1.0 1000.0 "10" "nvidia-tesla-p100" nil)
```

Comment:
Same comment about string gpu count.

Reply:
same response: gpus seem to show up as strings in the pod and node yaml

Reply:
This seems gross, but not unreasonable. @dposada do you have any thoughts about making these be integers? If nothing else, it makes the unit test a bit simpler without the count being a string argument.

Reply:
@scrosby we decided to make these integers because they are ultimately converted to a Quantity anyways.